### ISSUE TITLE
Slack web hook issue, looks like docs should be rewrited for dumb people as me

### ISSUE THREAD

**Issue by ashakiro** [secureCodeBox/secureCodeBox#1261]
cat myvalues.yaml
notificationChannels:
  - name: norules
    type: slack
    template: slack-messageCard
    skipNotificationOnZeroFinding: true
    endPoint: /home/ec2-user/notifications/values_slack_secrets.yaml
env:
    - name: myvalues
      valueFrom:
        secretKeyRef:
          name: myslacksecret
          key: SLACK_WEB_HOOK
          
cat values_slack_secrets.yaml
apiVersion: v1
kind: Secret
metadata:
    name: myslacksecret
type: Opaque
data:
    SLACK_WEB_HOOK: base64-url
    
kubectl apply -f values_slack_secrets.yaml
helm upgrade --install nwh secureCodeBox/notification-hook --values myvalues.yaml
kubectl apply -f /home/ec2-user/scans/nmap/test-nmap-scan.yaml

kubectl describe pods/notification-nmap-scanme.nmap.org-vrqz7-d8m6p
    State:          Waiting
      Reason:       CreateContainerConfigError
    Ready:          False
    Restart Count:  0
    Limits:
      cpu:     400m
      memory:  200Mi
    Requests:
      cpu:     200m
      memory:  100Mi
    Environment:
      SOME_ENV_KEY:  <set to the key 'some-key' in secret 'some-secret'>         Optional: false
      SMTP_CONFIG:   <set to the key 'smtp-config-key' in secret 'some-secret'>  Optional: false
      NAMESPACE:     default (v1:metadata.namespace)
      SCAN_NAME:     [nmap-scanme.nmap.org](http://nmap-scanme.nmap.org/)
      SOME_ENV_KEY:  <set to the key 'some-key' in secret 'some-secret'>         Optional: false
      SMTP_CONFIG:   <set to the key 'smtp-config-key' in secret 'some-secret'>  Optional: false
    Mounts:
      /home/app/config from notification-config (ro)
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-gpb6w (ro)
Conditions:
  Type              Status
  Initialized       True
  Ready             False
  ContainersReady   False
  PodScheduled      True
Volumes:
  notification-config:
    Type:      ConfigMap (a volume populated by a ConfigMap)
    Name:      notification-config
    Optional:  false
  kube-api-access-gpb6w:
    Type:                    Projected (a volume that contains injected data from multiple sources)
    TokenExpirationSeconds:  3607
    ConfigMapName:           kube-root-ca.crt
    ConfigMapOptional:       <nil>
    DownwardAPI:             true
QoS Class:                   Burstable
Node-Selectors:              <none>
Tolerations:                 [node.kubernetes.io/not-ready:NoExecute](http://node.kubernetes.io/not-ready:NoExecute) op=Exists for 300s
                             [node.kubernetes.io/unreachable:NoExecute](http://node.kubernetes.io/unreachable:NoExecute) op=Exists for 300s
Events:
  Type     Reason     Age                From               Message
  ----     ------     ----               ----               -------
  Normal   Scheduled  74s                default-scheduler  Successfully assigned default/notification-nmap-scanme.nmap.org-vrqz7-d8m6p to ip-*.us-east-2.compute.internal
  Normal   Pulled     12s (x8 over 73s)  kubelet            Container image "[docker.io/securecodebox/hook-notification:3.12.1](http://docker.io/securecodebox/hook-notification:3.12.1)" already present on machine
  Warning  Failed     12s (x8 over 73s)  kubelet            Error: secret "some-secret" not found
  
  Basically question: where am i wrong?

**Comment by J12934** [comment]
Hi @ashakiro 

its a bit hard to debug this as there files you pasted are formatted in a way which makes hard to tell exactly what was configured.

From context and the error at the bottom I'm thinking that you have a (possibly indentation) error in your `myvalues.yaml` config.

It should be formatted & configured like the following:

```yaml
notificationChannels:
  name: norules
  type: slack
  template: slack-messageCard
  skipNotificationOnZeroFinding: true
  # endpoint should equal the name of the env var its referencing
  endPoint: SLACK_WEB_HOOK

env:
  - name: SLACK_WEB_HOOK
    valueFrom:
      secretKeyRef:
        name: myslacksecret
        key: SLACK_WEB_HOOK
```

**Comment by ashakiro** [comment]
Hi @J12934 
I think that my problem is in 2 yaml files, and they don't see each other

![image](https://user-images.githubusercontent.com/14361753/177559022-5ea9f04b-84b3-4b64-841c-7181c3b3e71d.png)
